sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/repository/integration/AnnotatedQueryIT.java

```diff
@@ -280,4 +280,39 @@ public void testAnnotatedQueryWithMultipleCities() {
         final List<Address> resultsAsc2 = addressRepository.annotatedFindByCityIn(cities2, Sort.by(Sort.Direction.ASC, "postalCode"));
         assertAddressOrder(resultsAsc2, Address.TEST_ADDRESS2_PARTITION1, Address.TEST_ADDRESS1_PARTITION2, Address.TEST_ADDRESS1_PARTITION1);
     }
+
```

Comment:
@kushagraThapar unit tests are added in this file.

Reply:
Oh, got it, FYI, these are not unit tests These are integration tests :)